### ISSUE TITLE
JS error if an element with id of "language" exists in page.

### ISSUE THREAD

**Issue by bacanu** [KIProtect/klaro#93]
Chrome seems to add elements with ids to the window object, ex: `<div id="foo"></div>` is accesible using `window.foo`.

At src/utils/i18n.js:38  
`let lang = (window.language || document.documentElement.lang || 'en').toLowerCase()`  
it checks `window.language` and it assumes it will be a string, but if an element  
`<div id="language"></div>`  
exists in a page, it will throw an  
`Uncaught TypeError: (window.language || document.documentElement.lang || "en").toLowerCase is not a function` 

To reproduce this issue, replace /dist/index.html with: 
```
<!DOCTYPE html>
<html lang="EN-US">
    <head>
       
        <script type="application/javascript" src="config.js"></script>
        <script type="application/javascript" src="klaro.js"></script>
    </head>
    <body>
        <div id="language"></div>
    </body>
</html>
```

I'd make a PR, but I don't know how to fix this. Remove the `window.language` check? Check if it's string? Or if it has toLowerCase?

**Comment by jaller94** [comment]
A `typeof window.location === "string"` seems most appropriate in my understanding of the issue.

Thanks for reporting this and even offering to fix it. 👍

**Comment by jaller94** [comment]
Fixed by #94.
